.github/workflows/docs-update.yml

```diff
@@ -18,13 +18,12 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Setup uv
-        uses: astral-sh/setup-uv@v3
+      - uses: astral-sh/setup-uv@v3
         with:
           enable-cache: true
 
-      - name: install
-        run: uv sync --python 3.12 --group linting
+      - name: Install dependencies
+        run: uv sync --python 3.12 --group linting --all-extras
```

Comment:
`--all-extras` was missing and is the reason why CI fails on `main`.

Reply:
Good catch